disparate-flies/BangazonSite
# Order scaffolded basic templates

Please structure your pull requests this way:

- **Purpose of PR**: create blank controller and views for Order

- **Issue number this request impacts** : N/A

- **Description of what was changed**: Added scaffolded item to the Order Model

- **Thorough list of steps to test**

1.  Check out to my branch "orderScaffoldedBasic"
2.  Start Bangazon.sln
3. When this opens in Visual Studio, you should be the default OrdersController inside the Controllers folder
4. You should be able to click on the Views folder, Orders folder, and see default views for Index, Create, Details, Edit, and Delete.


if you don't do this you will be
**FIRED**
